api/v2/helpers/bouquets/bouquets_helper.py

```diff
@@ -9,3 +9,15 @@ def query_all_bouquets():
         bouquets = BouquetsSchema(many=True).dump(all_bouquets)
 
     return bouquets
+
+
+def create_bouquet(request_data):
+    """Function to add bouquet to database
+        :param request_data Required
+            Post data sent to server
+    """
+    BouquetsModel.add_bouquet(bouquet_name=request_data['bouquet_name'], refresh_url=request_data['refresh_url'],
```

Comment:
Here is the complexity I was talking about. It is not enough to add the record to the database. You need to test the credentials and API Keys passed to make sure they are good enough to make a connection to the Google Calendar API. We should not be storing invalid credentials. This will be the hard nut to crack.